org.ghidra_sre.Desktop.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Bastien Nocera <hadess@hadess.net> -->
+<component type="desktop">
+  <id>org.ghidra_sre.Desktop</id>
+  <name>Ghidra</name>
+  <summary>Ghidra Software Reverse Engineering Suite</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <!-- license from https://github.com/NationalSecurityAgency/ghidra/blob/master/LICENSE -->
+  <project_license>Apache-2.0</project_license>
+  <launchable type="desktop-id">org.ghidra_sre.Desktop.desktop</launchable>
+  <content_rating type="oars-1.1"/>
+  <description>
+    <p>
+      Ghidra is a software reverse engineering (SRE) framework. This framework includes
+      a suite of full-featured, high-end software analysis tools that enable users to
+      analyze compiled code.
+    </p>
+    <p>
+      Capabilities include disassembly, assembly, decompilation, graphing, and scripting,
+      along with hundreds of other features. Ghidra supports a wide variety of processor
+      instruction sets and executable formats and can be run in both user-interactive and
+      automated modes. Users may also develop their own Ghidra plug-in components and/or
+      scripts using Java or Python.
+    </p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by the National Security Agency Research Directorate.</p>
+  </description>
+  <url type="homepage">https://ghidra-sre.org/</url>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main Window</caption>
+      <image width="640" height="483">
+        https://raw.githubusercontent.com/flathub/com.adobe.Flash-Player-Projector/master/screenshots/flash-1.png
+      </image>
+    </screenshot>
+    <screenshot>
+      <caption>Playing “Samorost 2”</caption>
```

Comment:
Samorost?

Reply:
I'm going to be pretty angry if somebody asks me about screenshots again.
https://github.com/flathub/flathub/pull/1068#pullrequestreview-260087970